styles.css

```diff
@@ -31,20 +31,18 @@ main > section {
     margin: 0 auto;
 }
 
-section h3 {
-    margin-top: 100px;
+.imageContainer {
+    width: 750px;
```

Comment:
could consider making this a percentage so more responsive - but may distort images

Reply:
would also allow you to get two in a row when the screen is large, and reduce to one with a media query at smaller sizes

Reply:
I made the percentage change like you asked, it looks a lot better now thanks

Reply:
I would like to get two in a row but I don't think there's enough time and I was also don't think it's necessary to meet the requirements.